File: Data Structures & Algorithms/concatenation-of-array/submission-2.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        vector<int> newNums(2*size(nums));
        for(int i = 0; i<size(nums); i++) {
            newNums[i] = nums[i];
        }
        for(int i = 0; i<size(nums); i++) {
            newNums[size(nums)+i] = nums[i];
        }
        return newNums;
    }
};

File: Data Structures & Algorithms/concatenation-of-array/submission-2.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        ans = [None]*2*len(nums)
        for i in range(len(nums)):
            ans[i] = nums[i]
            ans[len(nums)+i] = nums[i]

        return ans
